pkg/reconciler/vertex/controller_test.go

```diff
@@ -169,7 +170,7 @@ func init() {
 
 func Test_NewReconciler(t *testing.T) {
```

Comment:
Vertex controller unit tests and Pipeline controller are independent. We should also add unit tests for event here even though we have similar ones for pipeline.